src/server/plugins/engine/services/formSubmissionService.js

```diff
@@ -0,0 +1,30 @@
+import { config } from '~/src/config/index.js'
+import { createLogger } from '~/src/server/common/helpers/logging/logger.js'
+import { postJson } from '~/src/server/services/httpService.js'
+
+const submissionUrl = config.get('submissionUrl')
+const logger = createLogger()
+
+/**
+ * Initiates a CDP file upload
+ * @param {{fileId: string, initiatedRetrievalKey: string}[]} files - batch of files to persist
+ * @param {string} persistedRetrievalKey - final retrieval key when submitting
+ */
+export async function persistFiles(files, persistedRetrievalKey) {
+  const postJsonByType = /** @type {typeof postJson} */ (postJson)
+
+  const payload = {
+    files,
+    persistedRetrievalKey
+  }
+
+  logger.info(`Completed persistence request`)
```

Comment:
This shouldn't be here

Reply:
Resolved with ffac4d1de8de2586f8473d8043ed987bdf09605f